Modix.Services/Mentions/MentionService.cs

```diff
@@ -33,14 +41,18 @@ internal class MentionService : IMentionService
 
         /// <inheritdoc />
         public async Task MentionRoleAsync(IRole role, IMessageChannel channel)
+            => await MentionRoleAsync(role, channel, string.Empty);
+
+        /// <inheritdoc />
+        public async Task MentionRoleAsync(IRole role, IMessageChannel channel, string message)
         {
             if (role is null)
                 throw new ArgumentNullException(nameof(role));
 
             if (channel is null)
                 throw new ArgumentNullException(nameof(channel));
 
-            if (role.IsMentionable)
+            if (role.IsMentionable && string.IsNullOrWhiteSpace(message))
```

Comment:
It feels a little odd to have differing functionality for `!@` based on whether there's message text:
![image](https://user-images.githubusercontent.com/2829282/56983469-b681e580-6b51-11e9-8d8f-0d083386c889.png)

Reply:
This was written with `!announce` in mind. If whatever role is defined as the announcement ping role (say `@News`) was marked as mentionable, then it would send two messages, the first saying "You can do that yourself" and the second being the actual announcement. Such messages would clutter the announcements channel and cause the staff member to have to clean up after the bot.

If it's preferred, I can just remove this test entirely.

Reply:
Restored additional functionality.